cpp/test/reverse_words_test.cpp

```diff
@@ -0,0 +1,39 @@
+#include "reverse_words.hpp"
+#include "gtest/gtest.h"
+
+TEST(reverse_words, do_not_modifies_empty_string) {
+    std::string s = "";
+    reverse_words(s);
+    EXPECT_EQ(s, "");
```

Comment:
By convention, the first argument should be the expected value and the second one - the actual